flink-rpc/flink-rpc-akka/src/test/java/org/apache/flink/runtime/rpc/akka/AkkaRpcActorTest.java

```diff
@@ -87,7 +87,10 @@
 
     @BeforeClass
     public static void setup() {
-        akkaRpcService = new TestingRpcService();
+        akkaRpcService =
+                new AkkaRpcService(
+                        AkkaUtils.createLocalActorSystem(new Configuration()),
+                        AkkaRpcServiceConfiguration.fromConfiguration(new Configuration()));
```

Comment:
nit: Could also use `AkkaRpcServiceConfiguration.defaultConfiguration()` to make it more explicit